app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb

```diff
@@ -33,6 +33,10 @@ def provider_object(connection = nil)
     (connection || connection_source.connect).api.credentials.find(manager_ref)
```

Comment:
What is expected here? ```manager_ref``` or ```native_ref```?

Reply:
that's fine because it's being build into an url

Reply:
Since that is talking to the Tower API I'm pretty sure it would be `native_ref`.